tests/drivers/spi/spi_loopback/boards/nrf54h20dk_nrf54h20_common.dtsi

```diff
@@ -4,6 +4,15 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#include <zephyr/dt-bindings/gpio/gpio.h>
```

Comment:
Not needed. It's included here: https://github.com/zephyrproject-rtos/zephyr/blob/3198db1b12292cf072b69f06000e9e258e529358/dts/vendor/nordic/nrf_common.dtsi#L8

Reply:
technically not needed I know, but it is good practice to include things only where needed, so if we start actually doing this in the future this sample is safe :D